firstvoices/backend/views/sites_views.py

```diff
@@ -213,3 +213,6 @@ def get_queryset(self):
             )
             .order_by(Upper("site__title"))
         )
+
+        queryset = queryset.exclude(Q(site__is_hidden=True) & Q(role__lte=Role.MEMBER))
```

Comment:
This code prevents hidden sites from appearing to members in both detail and list view